docs/pages/documentation/v6/usage-control.md

```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
+a policy restriction is detected. As soon as a `PolicyRestrictionException` is thrown, the access will be restricted.
+
+### Provide Access
+**Description:** This policy simply grants access to the resource. (Can be ignored)
+
+### Usage During Interval and Usage Until Deletion
+**Parameters:** start and end time (of type `ZonedDateTime`, a representation of an instant in the universal timeline)
+
+**Description:**
+Both the policies Usage During Interval and Usage Until Deletion use the same parameters and behave in the same way.
+They check if the data access time is between the start and end time defined and if that is not the case, a
+`PolicyRestrictionException` is thrown.
+The access time used is a ZonedDateTime, which represents an instant in the universal timeline, since it also contains
+date, time and zone information.
+The only difference is that the Usage Until Deletion rule *should* contain a postDuty field with a DELETE action.
+This is checked by a scheduled class called `ScheduledDataRemoval` and deletes the data after the interval has passed.
+The Usage Until Deletion policy should be used if it is desired that the resource be deleted after the time interval.
+
+### Duration Usage
+**Parameters:** a duration, as specified by the `Duration` java class.
+(Example: "PT10H" stands for 10 hours,
+[see more](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#abs--))
+
+**Description:** This policy starts a duration for a resource: the resource can only be accessed in the
+specified period. The duration starts counting from the artifacts' creation time.
+If the consumer tries to access the resource and the current time is over the allowed period a
+`PolicyRestrictionException` is thrown and the access is therefore denied.
+
+### Usage Logging
+**Prerequisites:** clearing house url in connector configuration
+
+**Description:** This defines the policy to send usage logs to the clearing house.
+The clearing house has to be defined in the connectors' configuration.
+The logs are sent as IDS Messages to the clearing house with the path "clearingHousePath/agreementId".
+The resource that is to be logged is sent in the payload.
+It is not possible to specify in the rule to which clearing house should be logged.
+If a clearing house is not specified or the message could not be delivered, the data access will still be granted.
+
+
+### N Times Usage
+**Parameters:** a  maximum number of accesses
+
+**Description:** This policy counts the access number of the resource and denies access if the
+access number is greater than the maximum number of accesses.
+It is recommended to disable automatic contract negotiation if you plan on using this policy, so that the
+data consumer does not negotiate a new contract once the maximum number of accesses has been reached.
+To disable automatic contract negotiation change the field to ``policy.negotiation=false`` in
+application.properties.
```

Comment:
Do we really want to recommend turning off negotiation completely?

Reply:
If there is no better workaround to avoid bypassing this policy, yes.